variables.tf

```diff
@@ -34,3 +34,13 @@ variable "use_calico_cni" {
   type    = bool
   default = false
 }
+
+variable "cluster_log_types" {
+  type       = list(string)
+  default    = ["api", "authenticator"]
+
+  validation {
+    condition      = length(setintersection(var.cluster_log_types, ["api", "audit", "authenticator", "controllerManager", "scheduler"])) > 0
```

Comment:
What if no log types are specified? `cluster_log_types == []`.

If the idea is to check that only valid log types are used, using set subtraction should work.